create-loggedout.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+mkdir ./login-build
+rm -rf ./login-build/*
+#rm -rf ./public/build.*.js
+#rm -rf ./public/vendor.*.js
+#rm -rf ./public/manifest.*.js
+
+echo Creating loggedout build
+./node_modules/.bin/webpack --config webpack.login.js --json | jq -r '.assets[] | .name'
```

Comment:
I had to install the `jq` package - something to add to the testing instructions.